subprojects/docs/src/docs/release/notes.md

```diff
@@ -172,9 +147,72 @@ Using a `FileCollection` provides a `FileCollectionDependency`.
 This allows Java and Kotlin to properly configure all types of dependencies
 and improves IDE support for the Groovy DSL.
 
+In addition, the Kotlin DSL now supports named arguments for external dependencies in this block:
+
+```kotlin
+testing {
+    suites {
+        val test by getting(JvmTestSuite::class) {
```

Comment:
We shouldn't be promoting the `by getting` pattern in the release notes for two reasons:
1. `getting` is the eager accessor
2. the `val ... by ...` idiom is considered unfriendly to novices
The novice friendly configuration idiom is:
```kotlin
    suites {
        "test"(JvmTestSuite::class) { ...
```